README.md

```diff
@@ -1,30 +1,96 @@
-* The errors are simple; do not over-think them.
-* Describe your approach in the ReadMe file's section below, as opposed to in-line comments.
+
+-   You will have 1 hour to clone the repo, create your own branch, and push your branch back to the remote repo.
+-   Please keep all work on your branch. Do not submit a pull request for main.
+-   The errors are simple; do not over-think them.
+-   Describe your approach in the ReadMe file's section below, as opposed to in-line comments.
 
 ### Instructions
-1. Clone the repo
-2. From `main`, create your own branch with `[your last name]/submission`
-3. Correct errors in the SQL and R script as follows:
-  * The SQL contains an error that will not generate correct output to be received by the R file. Please correct the SQL so that its output will match what the R file requires.
-  * The R file contains a single series of steps that could be separated into more descriptive functions or otherwise condensed to improve readability and maintainability. Please re-factor the R code to reflect cleaner, more maintainable code without altering functionality. *Please feel free to add separate function script files if you think this would help.*
-  * In the ReadMe file, modify the below section to explain your overall approach. Write as if speaking to a non-technical audience.
+
+1.  Clone the repo
+2.  From `main`, create your own branch with `[your last name]/submission`
+3.  Correct errors in the SQL and R script as follows:
+
+-   The SQL contains an error that will not generate correct output to be received by the R file. Please correct the SQL so that its output will match what the R file requires.
+-   The R file contains a single series of steps that could be separated into more descriptive functions or otherwise condensed to improve readability and maintainability. Please re-factor the R code to reflect cleaner, more maintainable code without altering functionality. *Please feel free to add separate function script files if you think this would help.*
+-   In the ReadMe file, modify the below section to explain your overall approach. Write as if speaking to a non-technical audience.
 
 **NOTE**: When making corrections in the SQL or R file, please also consider formatting adjustments that more closely conform to generally accepted style guidance (e.g., indentation, vertical alignment, etc.).
 
 **IMPORTANT: You should NOT need to create sample data and you do NOT need to compile your code to ensure it runs.** This is not a test of syntax, but of your ability to discover and streamline inefficiency in the data pipeline.
 
-4. Commit your edits and push your branch back to the remote repo within 1 hour
+4.  Commit your edits and push your branch back to the remote repo within 1 hour
 
 *GOOD LUCK!*
-  
-## About My Submission: `[Insert your First Name, Last Name]`
-Replace with your own narrative
+
+## USAREC Technical Skills Assessment
+
+### About My Submission: Lydia, Tanque
+
+**Project Objective**
```

Comment:
@agnlydtan I actually really like how you did the overview by treating it like a real DS project. But keep in mind the objective was not to perform data analysis--only to use the toy scenario to fix the errors.